ibis/expr/operations/temporal_windows.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from typing import Optional
+
+from public import public
+
+import ibis.expr.datatypes as dt
+from ibis.expr.operations.core import Column, Scalar  # noqa: TCH001
+from ibis.expr.operations.relations import Relation
+from ibis.expr.schema import Schema
+
+
+@public
+class WindowTVF(Relation):
```

Comment:
Perhaps `WindowingTVF` might be a better name?

And the subclassed ops would be: `TumbleWindowingTVF`, `HopWindowingTVF`, and `CumulateWindowingTVF`. This feels a bit better grammatically to me, and also ensures the tumble/hop/cumulate variants all have "window" in their name.

Reply:
Done!